build.gradle.kts

```diff
@@ -53,7 +53,7 @@ subprojects {
       googleJavaFormat()
     }
 
-    kotlin { ktfmt("0.30").googleStyle() }
+    kotlin { ktfmt("0.41").googleStyle() }
```

Comment:
This is the result of upgrading to Kotlin 1.7.20

Reply:
This also impacts the result of Kotlin formatter which causes a lot of file modifications in this PR.